docs/api/fhir-api-specification.md

```diff
@@ -3768,7 +3768,7 @@ On success, the server will update the existing resource given the id.
           "code": "207"
         }
       ],
-      "text": "Moderna COVID-19 Vaccine"
+      "text": "Moderna COVID-19 Vaccine (Non-US tradename: Spikevax)"
```

Comment:
On line 2543, a similar change is needed for an example of a resource that uses `Pfizer`.

Reply:
d375ee9